clickhouse/tests/docker/docker-compose.yaml

```diff
@@ -0,0 +1,185 @@
+    volumes:
+      - ./metrika.xml:/etc/metrika.xml
+      - ./macros/macros-03.xml:/etc/clickhouse-server/config.d/macros.xml
+      - ./users.xml:/etc/clickhouse-server/users.d/users.xml
+      - ./init.sql:/docker-entrypoint-initdb.d/init.sql
+    depends_on:
+      - clickhouse-zookeeper
+    networks:
+      - clickhouse-network
+
+  clickhouse-04:
+    image: yandex/clickhouse-server:${CLICKHOUSE_VERSION}
+    hostname: clickhouse-04
+    container_name: clickhouse-04
+    cap_add:
+      - IPC_LOCK
+      - NET_ADMIN
+    ulimits:
+      nproc: 65536
+      nofile:
+        soft: 262144
+        hard: 262144
+    ports:
+      - "8127:8123"
+      - "9004:9000"
+    volumes:
+      - ./metrika.xml:/etc/metrika.xml
+      - ./macros/macros-04.xml:/etc/clickhouse-server/config.d/macros.xml
+      - ./users.xml:/etc/clickhouse-server/users.d/users.xml
+      - ./init.sql:/docker-entrypoint-initdb.d/init.sql
+    depends_on:
+      - clickhouse-zookeeper
+    networks:
+      - clickhouse-network
+
+  clickhouse-05:
+    image: yandex/clickhouse-server:${CLICKHOUSE_VERSION}
+    hostname: clickhouse-05
+    container_name: clickhouse-05
+    cap_add:
+      - IPC_LOCK
+      - NET_ADMIN
+    ulimits:
+      nproc: 65536
+      nofile:
+        soft: 262144
+        hard: 262144
+    ports:
+      - "8128:8123"
+      - "9005:9000"
+    volumes:
+      - ./metrika.xml:/etc/metrika.xml
+      - ./macros/macros-05.xml:/etc/clickhouse-server/config.d/macros.xml
+      - ./users.xml:/etc/clickhouse-server/users.d/users.xml
+      - ./init.sql:/docker-entrypoint-initdb.d/init.sql
+    depends_on:
+      - clickhouse-zookeeper
+    networks:
+      - clickhouse-network
+
+  clickhouse-06:
+    image: yandex/clickhouse-server:${CLICKHOUSE_VERSION}
+    hostname: clickhouse-06
+    container_name: clickhouse-06
+    cap_add:
+      - IPC_LOCK
+      - NET_ADMIN
+    ulimits:
+      nproc: 65536
+      nofile:
+        soft: 262144
+        hard: 262144
+    ports:
+      - "8129:8123"
+      - "9006:9000"
+    volumes:
+      - ./metrika.xml:/etc/metrika.xml
+      - ./macros/macros-06.xml:/etc/clickhouse-server/config.d/macros.xml
+      - ./users.xml:/etc/clickhouse-server/users.d/users.xml
+      - ./init.sql:/docker-entrypoint-initdb.d/init.sql
+    depends_on:
+      - clickhouse-zookeeper
+    networks:
+      - clickhouse-network
+
+  clickhouse-zookeeper:
+    image: zookeeper:latest
+    hostname: clickhouse-zookeeper
+    container_name: clickhouse-zookeeper
+    ports:
+      - "2181:2181"
+      - "2182:2182"
+    networks:
+      - clickhouse-network
+
+  # clickhouse-client:
+  #   image: yandex/clickhouse-client:${CLICKHOUSE_VERSION}
+  #   entrypoint:
+  #     - /bin/sleep
+  #   command:
+  #     - infinity
+  #   depends_on:
+  #     - clickhouse-01
+  #     - clickhouse-02
+  #     - clickhouse-03
+  #     - clickhouse-04
+  #     - clickhouse-05
+  #     - clickhouse-06
+  #   networks:
+  #     - clickhouse-network
```

Comment:
Is this commented block necessary?

Reply:
For debugging or running arbitrary commands the client will be useful, especially for support

Reply:
Could you add a comment to indicate that? That will help people that need this for those cases.